### ISSUE TITLE
gfxrecon-info misspells VkStructureType

### ISSUE THREAD

**Issue by lunarpapillo** [LunarG/gfxreconstruct#586]
I ran across this using the SDK 1.2.182 version of `gfxrecon-info`:
```
$ which gfxrecon-info
/home/papillo/sdks/1.2.182.0/x86_64/bin/gfxrecon-info
$ gfxrecon-info --version
gfxrecon-info version info:
  GFXReconstruct Version 0.9.8 (10eb94f)
  Vulkan Header Version 1.2.182
$ gfxrecon-info /media/nas/smb_traces/vulkan/GFXR/tcubumes-mesa/vulkaninfo/vulkaninfo_20210928T101152.gfxr
[gfxrecon] ERROR - Failed to decode pNext value with unrecognized VkStructurType = 1000294001
[gfxrecon] ERROR - Failed to decode pNext value with unrecognized VkStructurType = 1000294001
File did not contain any frames
```
The top-of-`dev` `gfxrecon-info` does not provoke the same `pNext` error, but the source shows that `vkStructurType` is still misspelled in the generator:
```
$ git grep vkStructurType
framework/generated/generated_decode_pnext_struct.cpp:                GFXRECON_LOG_ERROR("Failed to decode pNext value with unrecognized VkStructurType = %d", (*sType));
framework/generated/vulkan_generators/decode_pnext_struct_generator.py:        write('                GFXRECON_LOG_ERROR("Failed to decode pNext value with unrecognized VkStructurType = %d", (*sType));', file=self.outFile)
```

**Comment by bradgrantham-lunarg** [comment]
This probably won't take more than 30 minutes to fix.  Let's squeeze out a little time and get this in.

**Comment by bradgrantham-lunarg** [comment]
Closed by #614
